wpilibNewCommands/src/main/native/include/frc2/command/ProxyScheduleCommand.h

```diff
@@ -33,9 +35,18 @@ class ProxyScheduleCommand
 
   explicit ProxyScheduleCommand(Command* toSchedule);
 
-  ProxyScheduleCommand(ProxyScheduleCommand&& other) = default;
+  /**
```

Comment:
nit: This constructor only takes in one command, but we refer to "schedules the given commands".

Reply:
agree; this should be fixed

Reply:
Fixed.